src/components/Login/InvalidUserMessage.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { makeStyles, Typography, Box } from '@material-ui/core';
+import { Warning } from '@material-ui/icons';
+
+const useStyles = makeStyles({
+  outlinedBox: {
+    width: '404px',
+    height: '104px',
+    border: '1px solid #AC3434',
+    boxSizing: 'border-box',
+    borderRadius: '10px',
+    paddingLeft: '26px',
+    paddingTop: '17px',
+    marginBottom: '61px',
+  },
+  warningIcon: {
+    width: '22.3px',
+    height: '19px',
+  },
+  invalidUserText: {
+    display: 'inline-block',
+    paddingLeft: '19.7px',
+    paddingBottom: '7px',
+  },
+});
+
+const InvalidUserMessage = () => {
```

Comment:
we can fold this component into `LoginPage` since it's the only place it'll be used anyways

Reply:
hmm yeah, but isn't that true for all components defined within a specific page's folder? e.g. `AddBinButton` will only be used for the Maps page, `StatusPills` will only be used for the PatientProfile page, etc. I think keeping this separate might be cleaner, since there's a bunch of custom styling for this component. 

Thoughts?